src/irvisualizer/StmtToViz_dependencies.template.html

```diff
@@ -0,0 +1,27 @@
+<!-- Bootstrap links -->
+<link href='https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css' rel='stylesheet' integrity='sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx' crossorigin='anonymous'>
```

Comment:
> I'd say these are quite reliable and used by millions of websites in production. However, using a CDN means users need an internet connection to view these files properly. It may be worthwhile to add local copies of js/css dependencies just to remove that requirement.

Agreed that would be worthwhile, but doesn't need to block this PR; I'd suggest opening an Issue here to suggest that as future work, so we don't forget about it.